<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaidServicesSetupDialog</class>
 <widget class="QDialog" name="PaidServicesSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>с:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CDateEdit" name="edtBegDate">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>по:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CDateEdit" name="edtEndDate">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbPayment">
        <property name="text">
         <string>Тип оплаты</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbPatient">
        <property name="text">
         <string>Пациентам</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbLab">
        <property name="text">
         <string>Лаборатории</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="CRBServiceComboBox" name="cmbService"/>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="cmbTypePayment">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>все</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>наличный</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>безналичный</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>по реквизитам</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="CRBComboBox" name="cmbLab"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblDate">
     <property name="text">
      <string>Дата:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="CClientComboBox" name="cmbPatient"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblTypePayment">
     <property name="text">
      <string>Тип оплаты:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblService">
     <property name="text">
      <string>В разрезе услуг:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblPatient">
     <property name="text">
      <string>В разрезе пациентов:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEventType">
     <property name="text">
      <string>Тип события:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="CRBComboBox" name="cmbEventType"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblLab">
     <property name="text">
      <string>По лаборатории:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblGroupBy">
     <property name="text">
      <string>Группировать по:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblCustomDateTime">
     <property name="text">
      <string>Отчёт составлен</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDateEdit" name="edtCustomDate"/>
   </item>
   <item row="7" column="2">
    <widget class="QTimeEdit" name="edtCustomTime"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CClientComboBox</class>
   <extends>QComboBox</extends>
   <header>library/ClientComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CRBServiceComboBox</class>
   <extends>QComboBox</extends>
   <header>RefBooks/Service/RBServiceComboBox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PaidServicesSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PaidServicesSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
